odoo/addons/base/views/ir_actions_views.xml

```diff
@@ -341,7 +341,7 @@
                         <separator string="Action Details" groups="base.group_no_one" />
                         <div class="d-flex flex-row flex-wrap gap-2" invisible="state != 'object_write'">
                             <field name="evaluation_type" class="oe_inline"/>
-                            <field name="update_path" widget="DynamicModelFieldSelectorChar" class="oe_inline" options="{'model': 'model_name'}"/>
+                            <field name="update_path" widget="field_selector" class="oe_inline" options="{'model': 'model_name'}"/>
```

Comment:
the renaming means an associated migration script is needed.
In my opinion, renaming is not strictly necessary (yet more elegant)